src/pages/components/SecondaryBar.qml

```diff
@@ -129,6 +135,14 @@ Column {
         }
     }
 
+    Notice {
+        id: pdfPrintingNotice
+        duration: 3000
+        //% "Already saving pdf"
+        text: qsTrId("sailfish_browser-la-already_printing_pdf")
+        verticalOffset: -Theme.itemSizeMedium
```

Comment:
```suggestion
        verticalOffset: -overlay.toolBar.toolsHeight * 4
```

Reply:
In order this to appear correctly above to the toolbar is should be like. This will both in portrait & landscape.